storage/paths/localstorage_cached.go

```diff
@@ -103,7 +104,9 @@ func (c *cachedLocalStorage) DiskUsage(path string) (int64, error) {
 		go func() {
 			du, err := c.base.DiskUsage(path)
 			if err != nil {
-				log.Errorw("error getting disk usage", "path", path, "error", err)
+				if !os.IsNotExist(err) {
```

Comment:
Maybe still debug-log the IsNotExist error?

Reply:
Based on the this code it seems like we actually do not care: https://github.com/filecoin-project/lotus/blob/a99946937c3ae82c9b3b89baeb5f883abad92eae/storage/paths/local.go#L83-L91

I think the reason for it, is that it can take quite a bit of time between the sector being reserved, and the file being transferred over.

I can add the same comment to this part of the code as well?

Reply:
It also seems from my testing that the defualt logging setting for `stores` is DEBUG, which makes this logging a bit unbearable. I will look into tuning that down in another PR.